src/navigation/actions.js

```diff
@@ -30,6 +34,26 @@ import { ROUTES } from './constants';
  *
  */
 
+export const goBack = () => dispatch => {
+  if (!RootNavigator.canGoBack()) BackHandler.exitApp();
+  else {
+    UIDatabase.write(() => {
+      const prescriptions = UIDatabase.objects('Prescription').filtered('status != "finalised"');
+      UIDatabase.delete('Transaction', prescriptions);
```

Comment:
But confused by this, is this the only side-effect performed on the `goBack` action?

Reply:
there are other pure effects which other reducers are acting on - main one is the `PagesReducer`